src/libs/DateUtils.js

```diff
@@ -113,16 +142,66 @@ function datetimeToCalendarTime(locale, datetime, includeTimeZone = false, curre
  * @returns {String}
  */
 function datetimeToRelative(locale, datetime) {
-    const date = getLocalMomentFromDatetime(locale, datetime);
+    const date = getLocalDateFromDatetime(locale, datetime);
+    return formatDistanceToNow(date);
+}
+
+/**
+ * Gets the zone abbreviation from the date
+ *
+ * e.g.
+ *
+ * PST
+ * EST
+ * GMT +07  -  For GMT timezone
+ *
+ * @param {String} datetime
+ * @param {String} selectedTimezone
+ *
+ * @returns {String}
+ */
+function getZoneAbbreviation(datetime, selectedTimezone) {
+    return formatInTimeZone(datetime, selectedTimezone, 'zzz');
+}
+
+/**
+ * Format date to a long date format with weekday
+ *
+ * @param {String} datetime
+ *
+ * @returns {String} Sunday, July 9, 2023
+ */
+function formatToLongDateWithWeekday(datetime) {
+    return format(new Date(datetime), CONST.DATE.LONG_DATE_FORMAT_WITH_WEEKDAY);
+}
 
-    return moment(date).fromNow();
+/**
+ * Format date to a weekday format
+ *
+ * @param {String} datetime
+ *
```

Comment:
it seems like the standard is not having this newline between `@param` and `@return`.